.gitignore

```diff
@@ -0,0 +1,104 @@
+# Logs
```

Comment:
Not really a fan of these auto-generated .gitignore files. But does not matter I guess :wink: